packages/report-server/src/reportBuilder/transform/functions/fetchData/query/QueryBuilder.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
```

Comment:
Mostly copied from the previous `packages/report-server/src/reportBuilder/query/QueryBuilder.ts`